tests/rptest/tests/scaling_up_test.py

```diff
@@ -88,3 +89,74 @@ def partitions_rebalanced():
         wait_until(partitions_rebalanced, timeout_sec=30, backoff_sec=1)
 
         self.run_validation(enable_idempotence=False, consumer_timeout_sec=45)
+
+    @cluster(num_nodes=8)
+    def test_adding_multiple_nodes_to_the_cluster(self):
+        group_topic_partitions = 16
+        self.redpanda = RedpandaService(self.test_context,
+                                        6,
+                                        extra_rp_conf={
+                                            "partition_autobalancing_mode":
+                                            "node_add",
+                                            "group_topic_partitions":
+                                            group_topic_partitions
+                                        })
+        # start single node cluster
+        self.redpanda.start(nodes=self.redpanda.nodes[0:3])
+        # create some topics
+        topics = []
+        total_replicas = 0
+        for _ in range(1, 5):
+            partitions = random.randint(10, 20)
+            spec = TopicSpec(partition_count=partitions, replication_factor=3)
+            total_replicas += partitions * 3
+            topics.append(spec)
+
+        total_replicas += group_topic_partitions * 3
+
+        for spec in topics:
+            DefaultClient(self.redpanda).create_topic(spec)
+            self.topic = spec.name
+        throughput = 100000 if not self.debug_mode else 1000
```

Comment:
Is there a known problem with debug mode and the verifiable producer? It seems strange to me to decrease the throughput for debug builds.

Reply:
i decreased the throughput as debug builds have massive overhead from ASAN and not using seastar allocator

Reply:
Ah that makes sense. Resolving this now.